lib/core/engine.js

```diff
@@ -132,7 +132,7 @@ class Engine {
   }
 
   pluginCommandService() {
-    this.registerModule('plugin_cmd', {embarkConfigFile: this.embarkConfig, packageFile: 'package.json'});
+    this.registerModule('plugin_cmd', {embarkConfigFile: this.embarkConfig, embarkConfig: this.config.embarkConfig, packageFile: 'package.json'});
```

Comment:
You can remove `embarkConfigFile` and `embarkConfig` from here, as they are accessible from `embark` inside the module.